### PR TITLE
Tracefoss 761 prepare for repos move

### PR DESCRIPTION
**Pull request by mkanal**
Prepare repos move

### CODE REVIEW

File: .github/workflows/chart-release.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ jobs:
       - name: Get helm charts current version
         id: step2
         run: |
-          chartVersion=helm-charts-$(cat ./charts/product-traceability-foss-frontend/Chart.yaml | grep "version:" | head -1 | cut -d ":" -d " " -f2)
+          chartVersion=helm-charts-$(cat ./charts/tx-traceability-foss-frontend/Chart.yaml | grep "version:" | head -1 | cut -d ":" -d " " -f2)
```
Comment (evia-fbedarf) [inline .github/workflows/chart-release.yml:35]: If you change the name of the folder here, you need to change the name of the folder in the project as well.

"project/charts/product-..." => "project/charts/tx-..."

Comment (mkanal) [reply .github/workflows/chart-release.yml:35]: Thanks i have fixed that.

**Review by evia-fbedarf — APPROVED** [review]
LGTM

**Comment by evia-fbedarf** [comment]
Will close this PR, because it is merged in the other project.

**Pull request closed without merging**